packages/blockchain-wallet-v4-frontend/src/data/analytics/reducers.js

```diff
@@ -8,6 +10,10 @@ const analytics = (state = INITIAL_STATE, action) => {
 
   switch (type) {
     case AT.LOG_EVENT: {
+      assert(path(['trackingData', 'length'], payload), 'trackingData cannot be empty')
+      const trackingAction = path(['trackingData', 1], payload)
+      assert(TRACKING_ACTIONS.indexOf(trackingAction) > -1, `tracking action: ${trackingAction} is not recognized`)
```

Comment:
Thanks, I prefer contains over indexOf

Reply:
Yeah, much nicer. 

Small note about `contains`: I took a quick peek at the docs and even though we use `contains` throughout the wallet, we should switch to `includes`. Looks like `contains` was deprecated since v0.26.0 (we're on 0.25.0). Anyway, change made.

https://ramdajs.com/docs/#contains
https://github.com/ramda/ramda/issues/2702